lib/trebuchet/strategy.rb

```diff
@@ -76,40 +77,69 @@ def self.name_for_class(klass)
     names[klass]
   end
 
-
-  ### Percentable module standardizes logic for percentage-based strategies
-
-  module Percentable
-
-    attr_reader :percentage
+  module PerDenominationable
```

Comment:
Definitely open to name suggestions. This is here to allow inheritance of `Percentable` - it is not advisable to remove `Percentable` altogether (and just in-line this directly into strategies) because we would lose backwards compatibility